.goreleaser.yaml

```diff
@@ -0,0 +1,74 @@
+builds:
+# clusterctl-aws
+- id: "clusterctl-aws"
+  main: ./cmd/clusterawsadm
+  binary: bin/clusterctl-aws
+  env:
+    - CGO_ENABLED=0
+  ldflags:
+  - -s -w 
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitMajor={{.Major}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitMinor={{.Minor}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitVersion={{.Version}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitCommit={{.Commit}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitTreeState={{.GitTreeState}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.buildDate={{.Date}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/cmd/clusterawsadm/cmd/version.CLIName=clusterctl-aws'
+  goos:
+  - linux
+  - darwin
+  - windows 
+  goarch:
+  - amd64
+  - arm64
+
+# clusterawsadm
+- id: "clusterawsadm"
+  main: ./cmd/clusterawsadm
+  binary: bin/clusterawsadm
+  env:
+    - CGO_ENABLED=0
+  ldflags:
+  - -s -w 
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitMajor={{.Major}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitMinor={{.Minor}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitVersion={{.Version}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitCommit={{.Commit}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitTreeState={{.GitTreeState}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.buildDate={{.Date}}'
+  goos:
+  - linux
+  - darwin
+  - windows 
+  goarch:
+  - amd64
+  - arm64
+
+archives:
+- id: clusterctl-aws
+  builds:
+  - clusterctl-aws
+  format_overrides:
+  - goos: windows
+    format: zip
+  name_template: "clusterctl-aws_{{ .Tag }}_{{ .Os }}_{{ .Arch }}"
+  format: binary
+- id: clusterawsadm
+  builds:
+  - clusterawsadm
+  format_overrides:
+  - goos: windows
+    format: zip
```

Comment:
We can drop this format override since we're uploading binaries now, not archives.